howtos/release-procedure.md

```diff
@@ -1,65 +1,48 @@
-# Release Procedure
+# Release & publish procedure
 
-## General
+The complete documentation is fully versioned. This means that there will be no static content that is the same across all versions. The documentation contains a drop down menu to set the version including `Next`, a version-less preview.
 
-The complete documentation is fully versioned. This means that there will be no static content that is the same across all versions. The documentation contains a drop down menu to set the version. The default is the latest version.
+## Perform a patch release
 
-## Roles
+Patch releases for docs can happen nearly any time, barring significant infrastructure changes. It's current best to ask for a release in #ask-documentation or include a release date in the PR.
```

Comment:
```suggestion
Patch releases for docs can happen nearly any time, barring significant infrastructure changes. It's current best practice to ask for a release in #ask-documentation or include a release date in the PR.
```

...or "currently best", but either way, the grammar is slightly off.

Reply:
Paging @christinaausley, because I agree something is off but maybe she'll want to rewrite the sentence entirely.

Reply:
"Patch releases for docs can happen nearly any time, barring significant infrastructure changes. Currently, it's recommended to either ask for a release in #ask-documentation or include a release date in the PR."